cmd/rpcdaemon/commands/eth_api.go

```diff
@@ -106,3 +113,32 @@ func (api *APIImpl) GetBlockTransactionCountByHash(ctx context.Context, blockHas
 	n := hexutil.Uint(len(block.Transactions()))
 	return &n, nil
 }
+
+func (api *APIImpl) GetTransactionByHash(ctx context.Context, hash common.Hash) (map[string]interface{}, error) {
+	tx, blockHash, blockNumber, txIndex := rawdb.ReadTransaction(api.dbReader, hash)
+
+	if tx == nil {
+		return nil, fmt.Errorf("transaction %#x not found", hash)
+	}
+
+	from := rawdb.ReadSenders(api.dbReader, blockHash, blockNumber)[txIndex]
+
+	v, r, s := tx.RawSignatureValues()
+
+	fields := map[string]interface{}{
```

Comment:
There is also `transactionIndex` field I see when I look at the `RPCTransaction` type in `internal/ethapi/api.go`